docs/GettingStarted.md

```diff
@@ -52,8 +44,8 @@ for your editor &mdash; they are hugely helpful when developing Go code.
 
 # Install DiscordGo
 
-Like any other Go package the fist step is to `go get` the package.  This will
-always pull the latest tagged release from the master branch. Then run 
+Like any other Go package, the first step is to `go get the package.  This will
```

Comment:
It again seems like you dropped the closing code tick.

Reply:
Good catch.